manifests/00-crd-extension-console-notification-banner.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  # TODO: consolenotificationbanners.console.openshift.io
+  # will we ever have consolenotifications that act like
+  # traditional notifications?
+  name: consolenotifications.console.openshift.io
+  annotations:
+    displayName: ConsoleNotification
```

Comment:
https://github.com/openshift/api/pull/318/files#diff-d02c06759975ba6309671b3248104d7bR10

`ConsoleNotificationBanner` vs `ConsoleNotification`.

Reply:
I suggest `ConsoleBanner`.

Reply:
Too generic?  

Though this one is still under consideration to get dropped & become `operator` config.

Reply:
`ConsoleNotification` seems to be the winner.